matrix-sdk-android/src/main/java/im/vector/matrix/android/api/session/room/timeline/TimelineEvent.kt

```diff
@@ -88,3 +89,15 @@ data class TimelineEvent(
  */
 fun TimelineEvent.getLastMessageContent(): MessageContent? = annotations?.editSummary?.aggregatedContent?.toModel()
         ?: root.getClearContent().toModel()
+
+
+fun TimelineEvent.getTextEditableContent(): String? {
+    val originalContent = root.getClearContent().toModel<MessageContent>() ?: return null
+    val isReply = originalContent.relatesTo?.inReplyTo != null
```

Comment:
This can maybe be a dedicated extension: `fun MessageContent.isReply()`

Reply:
done